### PR TITLE
Update PostgreSQL.xml

### PR DESCRIPTION
**Pull request by J0WI**
#15750

### CODE REVIEW

File: src/chrome/content/rules/PostgreSQL.xml [hunk 1]
```diff
@@ -1,49 +1,42 @@
...
+	<target host="archives.postgresql.org" />
+	<target host="babel.postgresql.org" />
+	<target host="buildfarm.postgresql.org" />
+	<target host="commitfest.postgresql.org" />
+	<target host="doxygen.postgresql.org" />
+	<target host="ftp.postgresql.org" />
+	<target host="git.postgresql.org" />
+	<target host="jdbc.postgresql.org" />
+	<target host="lists.postgresql.org" />
+	<target host="media.postgresql.org" />
```
Comment (Bisaloo) [inline src/chrome/content/rules/PostgreSQL.xml]: Can you add a test url please? http://media.postgresql.org/sfpug/instagram_sfpug.pdf

File: src/chrome/content/rules/PostgreSQL.xml [hunk 2]
```diff
@@ -53,16 +46,7 @@
 	<securecookie host="^\.postgresql.us$" name=".+" />
```
Comment (Bisaloo) [inline src/chrome/content/rules/PostgreSQL.xml]: This is not a target

File: src/chrome/content/rules/PostgreSQL.xml [hunk 3]
```diff
@@ -1,49 +1,42 @@
...
-
-
-	Problematic domains:
-
-		- postgresql.eu
-		- svn.postgresqlfr.org	(self-signed)
-		- postresql.org		(404)
-
 -->
 <ruleset name="PostgreSQL">
```
Comment (Bisaloo) [inline src/chrome/content/rules/PostgreSQL.xml]: Can you rename it to `PostgreSQL.org` and rename the file to `PostgreSQL.org.xml` please?

File: src/chrome/content/rules/PostgreSQL.xml (hunk 2 above)
Comment (J0WI) [reply src/chrome/content/rules/PostgreSQL.xml]: Fixed.

**Comment by Bisaloo** [comment]
Thank you!

**Pull request merged**